pyproject.toml

```diff
@@ -80,7 +79,7 @@ include = ["homeassistant*"]
 extend-exclude = "/generated/"
 
 [tool.pylint.MAIN]
-py-version = "3.10"
+py-version = "3.11"
```

Comment:
This will need a bit of work. Starting with `3.11`, `telnetlib` will be marked as deprecated. It's still used in a few places.
https://docs.python.org/3.11/library/telnetlib.html

A replacement could be https://telnetlib3.readthedocs.io/en/latest/ but it's probably better to add `pylint: disable` for now.

Reply:
telnetlib has still ~1.5 years of deprecation to go...